src/v/coproc/tests/wasm_event_tests.cc

```diff
@@ -95,8 +95,17 @@ SEASTAR_THREAD_TEST_CASE(verify_event_reconciliation) {
       .topics = {std::make_pair<>(
```

Comment:
I think you should split the tests out of this commit.

Reply:
I merged all things around wasm-event in one commit